reference/tools/data-migration/features/overview.md

```diff
@@ -20,15 +27,26 @@ The table routing feature enables DM to replicate a certain table of the upstrea
 ### Parameter configuration
 
 ```yaml
-routes:
+black-white-list:
```

Comment:
Warning: You should modify the "Parameter configuration" under the "Black & white table lists" section, not "Table routing" section. See https://github.com/pingcap/docs-cn/pull/2383/files. 

Please update accordingly. @TomShawn

Reply:
Updated, PTAL.